pageserver/src/walredo.rs

```diff
@@ -262,7 +262,7 @@ impl PostgresRedoManager {
             // next request will launch a new one.
             if let Err(e) = result.as_ref() {
                 error!(
-                    "error applying {} WAL records {}..{} ({} bytes) to base image with LSN {} to reconstruct page image at LSN {} n_attempts={}: {:?}",
+                    "error applying {} WAL records {}..{} ({} bytes) to key {key}, from base image with LSN {} to reconstruct page image at LSN {} n_attempts={}: {:?}",
```

Comment:
This log line is using string concatenation for historic reasons.

New stuff could also be added structurally:

```
error!(
    %key,
    ".....",
    /* million args */
);
```

This is a nit, I don't think we are matching this error in any queries.

Reply:
I had the same thought, but wanted to keep this change non-invasive and make it easier to search on historic incidences of errors here along with new ones.